### ISSUE TITLE
Check and update examples

### ISSUE THREAD

**Issue by JamieDanielson** [open-telemetry/opentelemetry-js-contrib#2528]
Review examples in examples directory and update packages as needed.

Add a comment for which example you are working on so others don't.

**Comment by sp6370** [comment]
Hey 👋🏽 !
I am working on the express example.

**Comment by xonev** [comment]
Hi there, I'll work on the web example.

**Comment by dsvanlani** [comment]
working on graphql example

**Comment by dsvanlani** [comment]
I took mongodb actually

**Comment by JamieDanielson** [comment]
> Currently that preference would likely be Jaeger v2 (supports direct OTLP). However, I'm hedging a bit because it doesn't support metrics and logs -- actually it might support metrics, not sure.

Yeah, I generally lean toward Jaeger v2 as well, but hesitate for the same reasons. It looks like it [supports OpenMetrics for metrics](https://github.com/jaegertracing/jaeger/blob/main/model/proto/metrics/README.md) (not OpenTelemetry) and no logs. Is it confusing to view tracing in Jaeger, with logs and metrics in the console? It seems like that may be the way to go...
